tikv/raftstore/peer_worker.go

```diff
@@ -250,6 +250,7 @@ func (rw *raftWorker) handleRaftReady(readyRes []*ReadyICPair) {
 			h.HandleRaftReady(&pair.Ready, pair.IC)
 		}
 		rw.handleReadyDc.collect(time.Since(begin))
+		metrics.PeerRaftProcessDuration.WithLabelValues("ready").Observe(time.Now().Sub(begin).Seconds())
```

Comment:
We can reuse the `time.Since(begin)` value at line 252.

Reply:
Updated.